_install-and-configure/install-opensearch/debian.md

```diff
@@ -0,0 +1,470 @@
+
+Generally speaking, installing OpenSearch from the Debian distribution can be broken down into a few steps:
+
+1. **Download and install OpenSearch.**
+   - Install manually from a Debian distribution or by creating a local repository.
+1. **Configure important system settings.**
+   - These settings are applied to the host before modifying any OpenSearch files.
+1. **(Optional) Test OpenSearch.**
+   - Confirm that OpenSearch is able to run before you apply any custom configuration.
+   - This can be done without any security (no password, no certificates) or with a demo security configuration that can be applied by a packaged script.
+1. **Configure OpenSearch for your environment.**
+   -  Apply basic settings to OpenSearch and start using it in your environment.
+
+The Debian distribution provides everything you need to run OpenSearch inside Debian-based Linux Distributions, such as Ubuntu.
+
+This guide assumes that you are comfortable working from the Linux command line interface (CLI). You should understand how to input commands, navigate between directories, and edit text files. Some example commands reference the `vi` text editor, but you may use any text editor available.
+{:.note}
+
+## Step 1: Download and install OpenSearch
+
+### Install OpenSearch from a package
+
+1. Download the Debian package for the desired version directly from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}. The Debian package can be downloaded for both **x64** and **arm64** architectures.
+1. Import the public GNU Privacy Guard (GPG) key. This key verifies that your OpenSearch package is signed.
+    ```bash
+    wget -qO - https://artifacts.opensearch.org/GPG-KEY-opensearch | sudo gpg --dearmor -o /usr/share/keyrings/opensearch-keyring.gpg
+    ```
+1. From the CLI, you can install the package with `dpkg` or `apt`.
+    **x64**
+    ```bash
+    # Install the x64 package using dpkg.
+    sudo dpkg -i opensearch-{{site.opensearch_version}}-linux-x64.deb
+    # Install the x64 package using apt.
+    sudo apt insall ./opensearch-{{site.opensearch_version}}-linux-x64.deb
+    ```
+    **arm64**
+    ```bash
+    # Install the arm64 package using dpkg.
+    sudo dpkg -i opensearch-{{site.opensearch_version}}-linux-arm64.deb
+    # Install the arm64 package using apt.
+    sudo apt install ./opensearch-{{site.opensearch_version}}-linux-arm64.deb
+    ```
+1. After the installation succeeds, enable OpenSearch as a service.
+    ```bash
+    sudo systemctl enable --now opensearch
+    ```
+1. Verify that OpenSearch launched correctly.
+    ```bash
+    sudo systemctl status opensearch
+    ```
+
+### Install OpenSearch from a local APT repository
+
```

Comment:
Moving the key import part from above to here.
Using the commands I provided in comments.